modules/installation/pages/install-proxy-uyuni.adoc

```diff
@@ -0,0 +1,48 @@
+[[install-proxy-uyuni]]
+= Install Uyuni Proxy with openSUSE
+
+{productname} Proxy can be installed on openSUSE.
+
+
+
+== Install Uyuni Proxy on openSUSE Leap 15.1
+
+.Procedure: Installing openSUSE Leap 15.1 with Uyuni
+
+. As the base system, install openSUSE Leap 15.1 with all available package updates applied.
+
+. Configure a resolvable fully quallified domain name (FQDN) with menu:yast[System > Netwok Settings > Hostname/DNS].
+
+. Add the repository for installing the {productname} Proxy software as [systemitem]``root``:
```

Comment:
Word order.

Reply:
more adjustments to be submitted separately.